Upgrade-Guide/op_version.md

```diff
@@ -0,0 +1,29 @@
+
+###op-version
+op-version is the operating version of the Gluster which is running.
+
+After Gluster upgrade, it is important that op-version is updated in all peers to avoid any 
+compatibility issues.
+
+###Updating op-version
+
+Once Gluster is upgraded, ensure that the op-version is also updated.
+
+    [root@~]#cat /var/lib/glusterd/vols/tv1/info  | grep ^op-version
+
+
+If this is not done automatically set op-version as below.
+For example, after upgrading to glusterfs-3.7.1, set op-version as:
+
+    [root@~]#gluster volume set all cluster.op-version 30701
+
+### Understanding op-version
+
+The op-version for a given X.Y.Z release will be an integer XYZ, with
```

Comment:
While the logic is correct but for every releases this may not be the case, If in any new release  no new features are been introduced or bug fixes are been down to take care of backward compatibility where op-version checks are used, there is no need of introducing a new op-version. For example, 3.7.9 had the max op-version as 30707 and this is why I am asking to have a metric for all our active releases with the op-version details and then this metric can be linked to the upgrade guide what we have. Does it make sense?